src/js/stores/AppFormTransforms.js

```diff
@@ -1,10 +1,14 @@
 const AppFormTransforms = {
+  cpus: (value) => parseFloat(value),
```

Comment:
should we think about a precision? I remember the usage of precision 4 on another place..